pipelines/cemba/cemba_methylcseq/README.md

```diff
@@ -1,40 +1,184 @@
-# CEMBA
-Analysis scripts for single nucleus methylation data (snmC-Seq) as part of the center for epigenomics of the mouse brain atlas (CEMBA)
-
-<ol>
-  <li>Automatic processing wrapper (CEMBA_automated_processing.pl)
-  <li>Gunzip wrapper (CEMBA_gunzip.pl)
-  <li>Demultiplex wrapper (CEMBA_demultiplex.pl & CEMBA_demultiplex_worker.pl)
-  <li>Cutadapt read trimming wrapper (cutadapt 1.12, CEMBA_indexed_fastq_trim.pl)
-  <li>Split pool for PBS submission (CEMBA_split_pool.pl)
-  <li>Bismark wrapper (CEMBA_bismark.pl, bismark v0.14.4)
-  <li>Remove clonal reads (picard tools 1.141)
-  <li>Reads filter -- for mouse data, just filter on MAPQ (samtools 1.3).
-  <li>Gzip demultiplexed fastq files (CEMBA_gzip.pl)
-  <li>Move fastq files for failed cells to recycle (CEMBA_fastq_filter.pl)
-  <li>Generate mapping summary (CEMBA_summary_dualSE_mapping.pl)
-  <li>Merge mapping summary (CEMBA_summary_dualSE_mapping_merge.pl)
-  <li>Methylpy mpileup and allc tables
-  <li>Create allc file (allc_SAMPLE.tsv.bgz) and corresponding tabix index (.tbi)
-  <li>Create binc files, genebody, promoter (Will be available in next methylpy release)
-</ol>
-
-## Submitting Workflows to CaaS
-
-To run workflows in the cloud, first generate an options file containing CaaS secrets.
-*DO NOT* check the generated file in.
-```bash
-$ ./scripts/inject-caas-secrets <path/to/options.json> > <path/not/in/repo/options.caas.json>
-```
-
-Then submit your workflow with the generated options in one of two ways:
-
-### Manually via Swagger
-
-Follow the instructions [here](https://docs.google.com/document/d/1od3J2IgO5z27BKh8DEP4lbPilDE1t6f4xk8DdvkxGmI/edit#).
-When told to upload options JSON, choose to upload the options file generated by `inject-caas-secrets`.
-
-### Via the CLI
-
-TODO after the [`cromwell-tools` refactor](https://github.com/broadinstitute/cromwell-tools/pull/43) exposes CaaS-related
-options on the CLI.
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [CEMBA_v1.0](CEMBA.wdl) | July 28, 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
```

Comment:
I always get confused about when to use the name CEMBA and when to use snmc-seq2 like Hanqing mentioned in his email. Tim would probably know better than I, perhaps you already checked with him.

Reply:
Tim and I didn't discuss, but I can ask him. I assumed we chose the name or used the workflow name (CEMBA). I thought the snmC-seq2 was the name of the biological assay they developed, not necessarily the computational pipeline.

Reply:
The plan is to update the name on a future ticket.